### PR TITLE
feat(D5, Extension Example modules) :: Refactor Parent-Child module

### PR DESCRIPTION
**Pull request by itowhid06**
Fixes: https://github.com/elegantthemes/Divi/issues/33769
Fixes: https://github.com/elegantthemes/Divi/issues/33770

### CODE REVIEW

File: src/components/parent-module/settings-advanced.tsx [hunk 1]
```diff
@@ -3,28 +3,28 @@ import React, { ReactElement } from 'react';
...
       cssFields={cssFields}
     />
-    <Attributes />
-    <Visibility />
-    <Transition />
+    <AttributesGroup />
+    <VisibilitySettingsGroup />
+    <TransitionGroup />
     <PositionGroup />
     <ScrollGroup />
```
Comment (sofyansitorus) [inline src/components/parent-module/settings-advanced.tsx:29]: I think this should be `ScrollSettingsGroup`

**Review by sofyansitorus — CHANGES_REQUESTED** [review]
@itowhid06 I have several feedback.

File: src/components/child-module/styles.tsx [hunk 2]
```diff
@@ -5,132 +5,82 @@ import React, { ReactElement } from 'react';
...
+      {/* Module */}
+      {elements.style({
+        attrName: 'module',
+        styleProps: {
+          disabledOn: {
+            disabledModuleVisibility: settings?.disabledModuleVisibility,
+          },
+        },
+      })}
       <TextStyle
```
[the remark this answers is not in the document]
Comment (itowhid06) [reply src/components/child-module/styles.tsx:48]: @sofyansitorus I didn't get this properly. Can you please elaborate a bit?

[the remark this answers is not in the document]
Comment (sofyansitorus) [reply src/components/child-module/styles.tsx:48]: Do we still need to use this block?

```
<TextStyle
        selector={contentContainerSelector}
        attr={attrs?.module?.advanced?.text}
      />
```

**Review by sofyansitorus — APPROVED** [review]
LGTM now

**Pull request merged**